core/src/main/kotlin/xtdb/cache/MemoryCache.kt

```diff
@@ -25,6 +27,7 @@ class MemoryCache
 
     @Suppress("MemberVisibilityCanBePrivate")
     val maxSizeBytes: Long,
+    val meterRegistry: MeterRegistry,
```

Comment:
Wasn't sure about arg ordering here, this seemed most sensible to me given optionality of pathLoader and non-optionality of meterRegistry